android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -386,6 +382,16 @@ private void downloadPageId() {
                     });
     }
 
+    private void storePageName(String link, String key) {
+        if(SharedPreferencesUtil.getString(key, null) == null) {
+            RealmList<SocialLink> socialLinks = event.getSocialLinks();
+            RealmResults<SocialLink> page = socialLinks.where().equalTo("name", link).findAll();
+            SocialLink socialLink = page.get(0);
+            String pageName = socialLink.getLink().substring(socialLink.getLink().lastIndexOf('/') + 1);
```

Comment:
lastIndexOf('/'" will create problem. What if link is like `https://www.facebook.com/pageId/`? You can use `facebook.com/` instead of `/`